lib/appsignal/utils/map_filter.ex

```diff
@@ -1,4 +1,7 @@
 defmodule Appsignal.Utils.MapFilter do
+  # TO-DO: Move this module to the Phoenix package, affter
```

Comment:
```suggestion
  # TODO: Move this module to the Phoenix package, after
```
It's more common to write it like this and easier to find back. I only look for "TODO" or "todo".